datasets/plugins/executors/metaflow_executor.py

```diff
@@ -0,0 +1,28 @@
+from datasets.context import Context
+from datasets.dataset_plugin import ProgramExecutor
+
+
+class MetaflowExecutor(ProgramExecutor):
```

Comment:
To play devils advocate here, if `metaflow` plans at some point to open source its online serving pieces how would this affect the naming conventions used here? I guess I'm asking if we need to think ahead and call this something like `MetaflowBatchExecutor` or if that's overkill.

Reply:
Most likely the online executor would fit within the same context, and this `MetaflowExecutor` would continue to use `from metaflow import current` ...

Reply:
Cool, will resolve this thread then if you're not concerned at this time 😄